_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-modernist
```

Comment:
Please don't change the theme - it is intentionally kept simple and neutral. Themes are always subjective so I'm not open to discuss theme changes.